docs/language/functions.rst

```diff
@@ -279,6 +279,38 @@ constructor.
     for storage rent and there is a minimum deposit needed for the contract. As a result,
     constructors always receive value on Parity Substrate.
 
+Overriding function selector
+____________________________
+
+When a function is called, the function selector and the arguments to the function
+are serialized (also known as abi encoded) and passed to the program. The selector
+is what the runtime program uses to determine what function was called. Usually
+the function selector is generated from a deterministic hash value of the function
+name and its arguments.
+
+The selector value can be overriden with the ``selector=hex"deadbea1"`` syntax,
+for example:
+
+.. code-block:: solidity
+
+    contract foo {
+        function get_foo() selector=hex"01020304" public return (int) {
```

Comment:
Does the position of the selector overwrite matter? I'd rather move the selector overwrite at the beginning or at the end of the function declaration.

Reply:
The selector is part of the function attributes, which can occur in any order after the parameters and before the `returns (..)` (if any).